fe/fe-core/src/main/java/com/starrocks/external/hive/HiveRepository.java

```diff
@@ -66,11 +67,16 @@ public HiveMetaClient getClient(String resourceName) throws DdlException {
             if (resource == null) {
```

Comment:
If this is a common class, we need move it to another place

Reply:
Maybe need but not now, actually we need to distinguish the corresponding implementations of metastore functions and table functions in the codes later.